### PR TITLE
BS-13332 throw an exception when exporting a bar with empty internal mapping

### PR DESCRIPTION
**Pull request by rbioteau**
https://github.com/bonitasoft/bonita-studio-sp/pull/23

### CODE REVIEW

File: bundles/plugins/org.bonitasoft.studio.designer/src-test/java/org/bonitasoft/studio/designer/core/bar/FormMappingBarResourceProviderTest.java [hunk 1]
```diff
@@ -62,8 +69,16 @@
...
     @InjectMocks
-    private FormBarResourceProvider formMappingBarResourceProvider;
+    private FormMappingBarResourceProvider formMappingBarResourceProvider;
+
+    @Before
+    public void setUp() throws Exception {
+        when(preferenceStore.getBoolean(DesignerPreferenceConstants.FORCE_INTERNAL_FORM_MAPPING, true)).thenReturn(false);
+    }
 
     @Test(expected = NullPointerException.class)
```
Comment (apupier) [inline bundles/plugins/org.bonitasoft.studio.designer/src-test/java/org/bonitasoft/studio/designer/core/bar/FormMappingBarResourceProviderTest.java]: We really expect a NPE?? Why don't we handle this use cas emore properly?

**Comment by apupier** [comment]
please provide JIRa number in commit message

**Comment by rbioteau** [comment]
Can we edir commit message ?

**Comment by rbioteau** [comment]
git commit --amend -m "New message" will update latest commit message

**Comment by apupier** [comment]
we can edit the message directly in Eclipse with one th elatest version of EGit, just by right-clicking on th ecommit in history view "Modify--> Reword"

**Pull request merged**